test/controllers/users.js

```diff
@@ -99,5 +118,44 @@ describe('users controller', () => {
         });
     });
   });
+  describe('/users GET', () => {
```

Comment:
You could also add some test cases where you play around with the limit and offset params.